### PR TITLE
Variables

### PR DESCRIPTION
**Pull request by dkroderos**

### CODE REVIEW

File: src/main/Common/Constants.cs
Comment (dkroderos) [inline src/main/Common/Constants.cs:1]: I would like to seek help about double checking these environment variables. I'm also planning to change this into a partial class to separate the constants.

File: src/main/IO/Process/Services/Avatars/AvatarSettingsRepository.cs [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+
+            var index = line.IndexOf('=');
+
+            // If no "=" found, go to the next line
+            if (index == -1) continue;
+
+            var key = line[..index].Trim();
+            var value = line[(index + 1)..].Trim();
+
+            switch (key.ToUpper())
```
Comment (dkroderos) [inline src/main/IO/Process/Services/Avatars/AvatarSettingsRepository.cs:45]: I'm looking for a way to shorten these switch statements.

File: src/main/Common/Constants.cs
Comment (elmerbool) [reply src/main/Common/Constants.cs:1]: Please create a ticket for this in Taiga with references to the classes and files involved.

File: src/main/IO/Process/Services/Avatars/AvatarSettingsRepository.cs (hunk 1 above)
Comment (elmerbool) [reply src/main/IO/Process/Services/Avatars/AvatarSettingsRepository.cs:45]: Please create a ticket for this in Taiga with references to the classes and files involved.

**Review by elmerbool — APPROVED** [review]
Thanks for the changes David. Please look into the comments I posted.

**Pull request merged**